torch/csrc/jit/passes/specialize_autogradzero.cpp

```diff
@@ -79,6 +150,25 @@ struct AutogradZeroSpecializer {
     }
   }
 
+  static Node* getUseWithAttribute(Value* inp, Symbol kind) {
```

Comment:
nit: `getUseWithAttribute` doesn't fully reflect the implementation, since we're also filtering out non `profile_ivalue` uses